docs/en/getting-started/install.md

```diff
@@ -1,13 +1,34 @@
 ---
-sidebar_label: Installation
-sidebar_position: 1
-keywords: [clickhouse, install, installation, docs]
-description: ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture.
-slug: /en/getting-started/install
-title: Installation
+sidebar_label: Install
+keywords: [clickhouse, install, getting started, quick start]
+slug: /en/install
```

Comment:
does it mean that it won't be available by the old path?

Reply:
the PR in the clickhouse-docs repo adds the redirects, so install.md, and requirements.md links both go the same place.  No old links will break.

Reply:
this is the other PR: https://github.com/ClickHouse/clickhouse-docs/pull/438